### PR TITLE
Datavault bulk predictor

### PR DESCRIPTION
**Pull request by chinmaygadkari7**
Changes include:
1. Addded `/prediction` endpoint in ilabs Api
2. Modified list collection code to find collections recursively. 
3. New datavault predictor script
4. New datavault bulk predictor

### CODE REVIEW

File: ilabs/client/ilabs_api.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ def __init__(self, user_key=None, timeout=None, user_agent=None):
         self.URL_FEEDBACK = self.URL_API_BASE + '/documents/training/{domain}/'
 
         self.URL_PREDICT  = self.URL_API_BASE + '/reference/{domain}/{name}'
+        self.URL_PREDICT_ = self.URL_API_BASE + '/prediction/{domain}/{collection}/{name}'
```
Comment (mkroutikov) [inline ilabs/client/ilabs_api.py]: better name, please. Maybe `PREDICT_DATAVAULT`, not `PREDICT_`

File: ilabs/client/test/test_ilabs_api.py [hunk 2]
```diff
@@ -18,13 +18,14 @@ def test_request(self):
         with mock.patch('ilabs.client.ilabs_api.send_request', mock_request) as request:
 
             api._request('OPTIONS', 'http://www.gogole.com', b'some content',
-                content_type='test/test')
+                content_type='test/test', query=None)
```
Comment (mkroutikov) [inline ilabs/client/test/test_ilabs_api.py]: not needed change, as query=None is the default

**Review by mkroutikov — CHANGES_REQUESTED** [review]
Minor

**Review by mkroutikov — DISMISSED** [review]
Looks good

**Comment by chinmaygadkari7** [comment]
@mkroutikov Version bump to 0.2.6?

**Comment by mkroutikov** [comment]
Yes, please. After this is merged, please kill the obsolete PR and branches

**Review by mkroutikov — APPROVED** [review]
:)

**Pull request merged**